types/hapi-auth-jwt2/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-empty-interface": false
```

Comment:
Don't disable lint rules.

Reply:
The reason for this is to to allow user enhancements.

Reply:
That makes sense, ignore this comments.